### PR TITLE
fixed create company users link style

### PR DESCRIPTION
**Pull request by jfkonecn**
fixes oncokb/oncokb#3728

https://github.com/oncokb/oncokb-public/assets/24961694/54dd0268-d5ca-434e-8f7b-dca14c6a4b14

### CODE REVIEW

File: src/main/webapp/app/pages/companyPage/CompanyPage.tsx [hunk 1]
```diff
@@ -516,13 +516,12 @@ export default class CompanyPage extends React.Component<ICompanyPage> {
...
                         <div>
-                          <QuickToolButton>
-                            <Link
-                              to={`/companies/${this.company.id}/create-users`}
-                            >
-                              Create Company Users
-                            </Link>
-                          </QuickToolButton>
+                          <Link
+                            className="btn btn-outline-primary"
```
Comment (calvinlu3) [inline src/main/webapp/app/pages/companyPage/CompanyPage.tsx:520]: This button is missing the `m-2` that is used in QuickToolButton. I think we should probably still use that component. To link to another page, you can use the `routing` prop to change the history. See the UserPage.tsx file for an example.

Comment (calvinlu3) [reply src/main/webapp/app/pages/companyPage/CompanyPage.tsx:520]: I think the missing margin is also why the Screenshot Test is also failing.

Comment (jfkonecn) [reply src/main/webapp/app/pages/companyPage/CompanyPage.tsx]: I tried using the history prop, but it's breaking the back button.
You not have to hit it twice to get the browser to go back.
Are you sure we need it?

Comment (calvinlu3) [reply src/main/webapp/app/pages/companyPage/CompanyPage.tsx]: Oh that's interesting. Have you tried just using the `QuickButtonTool` with an onClick. Maybe adding the history prop call with the `Link` component is causing some issues?

Comment (calvinlu3) [reply src/main/webapp/app/pages/companyPage/CompanyPage.tsx]: If that doesn't work, I think the solution you have now is good too.

Comment (jfkonecn) [reply src/main/webapp/app/pages/companyPage/CompanyPage.tsx]: What's it for? Shouldn't the like handle the routing history?

Comment (calvinlu3) [reply src/main/webapp/app/pages/companyPage/CompanyPage.tsx]: I imagine the QuickToolButton was created so the styling can be reused for all quick tools.

The Link can be used to handle redirecting. I believe the implementation of that component probably pushes to the history to change the location. We can also update the history by directly calling `history.push()`.

I don't see a problem with either approach:
a. Copy quick tool button styles to Link component
b. Or using history to redirect in QuickToolButton component

**Comment by jfkonecn** [comment]
@zhx828, I want to talk to you about using QuickToolButton for links when you get back.

**Pull request merged**